src/main/java/edu/ie3/tools/models/persistence/ICONWeatherModel.java

```diff
@@ -21,8 +21,8 @@
   private static final long serialVersionUID = -3506091597737302833L;
 
   @Id
-  @Column(name = "datum", nullable = false)
-  private ZonedDateTime date;
+  @Column(name = "time", nullable = false)
+  private ZonedDateTime time;
 
   @Id
   @ManyToOne(cascade = CascadeType.DETACH)
```

Comment:
Don't know how you feel about this. As the many-to-one relation has no `@JoinColumns` annotation, it defaults back to `coordinate_id`, as this field ist named coordinate and the database id of `CoordinateModel` is `id`. However, this is quite implicit and I would like it to be more explicit.